cmd/lintwalk/main.go

```diff
@@ -27,39 +52,61 @@ func Main() {
 	}
 	srcRoot = filepath.Clean(srcRoot)
 
+	srcRoot, err := filepath.Abs(srcRoot)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	excludeRE, err := regexp.Compile(*exclude)
+
+	testRE := regexp.MustCompile(".*_test")
+
 	if err != nil {
-		log.Fatalf("bad -exclude pattern: %v", err)
+		log.Fatal(err)
```

Comment:
Hmm, I wonder why you think it's better to exclude error context from `log.Fatal` message.